scripts/cdnify.js

```diff
@@ -200,7 +200,14 @@ const info = async (name : string) : Promise<PackageInfo> => {
     if (!json['dist-tags'] || !json['dist-tags'][options.disttag]) {
         throw new Error(`${ options.disttag } dist tag not defined`);
     }
-    
+
+    const localPackage = await getPackage();
+    const publicRegistryVersion = json['dist-tags'][options.disttag];
+
+    if (options.disttag === 'latest' && localPackage.version !== publicRegistryVersion) {
```

Comment:
When releasing the JS SDK, I've hit a problem where `npm run activate` doesn't push the latest release to the cdn. And I have to try running `npm run cdnify` manually until it gets the correct version of the public npm registry. It seems the the npm public registry can return mixed results. Maybe due to their edge cache 🤔